### PR TITLE
Hotfix/1.5.2

### PR DESCRIPTION
**Pull request by rick-stevens**

### CODE REVIEW

File: postBuild.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
-const pkg = require('./package.json');
-const fs = require('fs');
+const pkg = require('./package.json')
```
Comment (codeorelse) [inline postBuild.js]: What's the reason you removed the ;?

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@bluemango/slp-helper",
-  "version": "1.5.1",
+  "version": "1.5.2",
   "description": "This library gives you shortcuts to develop Smart LemonPI Pixels. It also acts as an active value watcher, so you don't need to build in existence checks, or wrap timeouts and intervals.",
   "main": "src/slp",
   "scripts": {
-    "start": "eslint src/slp.js && webpack && node postBuild.js",
-    "lint": "eslint src/slp.js"
+    "start": "standard && webpack && node postBuild.js",
```
Comment (codeorelse) [inline package.json]: Why do we change to `standard` here?

Comment (rick-stevens) [reply package.json]: Purely for research

File: src/slp.js [hunk 3]
```diff
@@ -306,11 +303,26 @@ window.lemonpi = window.lemonpi || [];
     if (hashedResult !== lastScrapedHash) {
       lastScrapedHash = hashedResult;
 
+      // Use the propSeen type as default
+      if (!result.type) {
+        result.type = 'propSeen';
```
Comment (codeorelse) [inline src/slp.js]: YES!

File: postBuild.js (hunk 1 above)
Comment (codeorelse) [reply postBuild.js]: Still applies.

Comment (rick-stevens) [reply postBuild.js]: Forgot!

**Pull request merged**